.github/workflows/test.yml

```
@@ -0,0 +1,74 @@
+                        phpunit: '10.*'
+                    -   laravel: '13.*'
+                        phpunit: '11.*'
+        name: php ${{ matrix.php }} using Laravel ${{ matrix.laravel }} and PHPUnit ${{ matrix.phpunit }}
+        runs-on: ${{ matrix.operating-system }}
+        steps:
+            -   name: Checkout
+                uses: actions/checkout@v6
+            -   name: Setup PHP
+                uses: shivammathur/setup-php@v2
+                with:
+                    php-version: ${{ matrix.php-version }}
```

actionlint (1.7.12)

[error] property "php-version" is not defined in object type {laravel: string; operating-system: string; php: number; phpunit: string}

(expression)